.github/workflows/playwright.yml

```diff
@@ -59,18 +59,29 @@ jobs:
     # - name: Print server logs
     #   if: failure()
     #   run: cat ${{ github.workspace }}/server.log
+
     - name: Install dependencies
       run: |
         cd ${{ github.workspace }}/packages/near-fast-auth-signer-e2e-tests
         npm install -g yarn && yarn
     - name: Install Playwright Browsers
       run: yarn playwright install --with-deps
+    # This .env variables will be available to be used in the signer app
+    - name: Send secrets to process.env
+      run: |
+        echo "ETH_PROVIDER_URL_TESTNET=${{ secrets.ETH_PROVIDER_URL_TESTNET }}" >> $GITHUB_ENV
+        echo "BNB_PROVIDER_URL_TESTNET=${{ secrets.BNB_PROVIDER_URL_TESTNET }}" >> $GITHUB_ENV
+        echo "BTC_PROVIDER_URL_TESTNET=${{ secrets.BTC_PROVIDER_URL_TESTNET }}" >> $GITHUB_ENV
```

Comment:
why does it have to be passed down to .env instead of directly feed into process.env like 
`MULTICHAIN_TEST_ACCOUNT_ID: ${{ secrets.MULTICHAIN_TEST_ACCOUNT_ID }}`

Reply:
@hcho112 that's right. 
Fixed.